src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -100,31 +104,39 @@ const ProfileHeader = ({
         >
           {showBadgeImage && (
             <img
-              className="profile-service-badge"
               src={SERVICE_BADGE_IMAGE_PATHS.get(latestBranchOfService)}
-              alt="service badge"
+              alt={`${latestBranchOfService} seal`}
+              className="profile-service-badge"
+              aria-hidden="true"
+              role="presentation"
             />
           )}
         </div>
         <div className="name-and-title-wrapper">
           <h1 className={[...titleClasses, ...titleClassesMedium].join(' ')}>
             Your Profile
           </h1>
-          <h2
-            className={[...fullNameClasses, ...fullNameClassesMedium].join(' ')}
-          >
-            {fullName}
-          </h2>
-          {latestBranchOfService && (
-            <h3
-              className={[
-                ...latestBranchClasses,
-                ...latestBranchClassesMedium,
-              ].join(' ')}
+          <dl className="vads-u-margin-y--0">
+            <dt className="sr-only">Name: </dt>
+            <dd
+              className={[...fullNameClasses, ...fullNameClassesMedium].join(
+                ' ',
+              )}
```

Comment:
If we could join these classes outside the JSX it could save the awkward template string wrapping doing the line breaks

Reply:
True. I'm not sure that doing this spread here is the best way to handle this but it was the pattern I'd been using so I stuck with it.